rules/windows/process_access/proc_access_win_lsass_remote_access_trough_winrm.yml

```diff
@@ -1,12 +1,12 @@
-title: Mimikatz through Windows Remote Management
+title: Remote Lsass Process Access Through Windows Remote Management
 id: aa35a627-33fb-4d04-a165-d33b4afca3e8
 status: stable
-description: Detects usage of mimikatz through WinRM protocol by monitoring access to lsass process by wsmprovhost.exe.
+description: Detects remote access to the lsass process via WinRM. Could be a sign of credential dumping from tools such as mimikatz.
```

Comment:
```suggestion
description: Detects remote access to the LSASS process via WinRM. This could be a sign of credential dumping from tools like mimikatz.
```